docs/beta-scenarios.md

```diff
@@ -0,0 +1,103 @@
+# Beta Testing Connectors in the Extension Gallery  
+-------
+
+Welcome to the Beta of the Connectors in the Extension Gallery. Tableau has worked closely with a number of its Technology Partners to create connectors for their platform. These connector plugins will replace having to connect to your data using the "Other ODBC/JDBC" connector, and provide a smooth and performant connection to these databases. Installing the new connector only takes a few minutes and you'll be able to see the difference for yourself.
+We've provided a few test scenarios below, which may take between 10 and 20 minutes to run through to ensure the connector will work with your environment.
+
+Once you've downloaded a connector and tried these scenarios we'd love to hear from you about your experience! **Please submit feedback to us through the connector-specific feedback form found on the [Extension Gallery](https://extensiongallery.tableau.com/connectors).** Thanks for  participating!
+
+### Gaining access to Beta Connectors in the Gallery:
+* Go to https://extensiongallery.tableau.com/
+On the top right hand corner of the screen, click 'Sign In' and follow the sign in process using your tableau.com login credentials.
+* Click on your user icon in the top right hand corner and select 'Sign up for Connectors Beta'.
+* Click the 'Sign Up' button (the page will automatically refresh and you will now see any Beta connectors available for testing)  
+
+### Notes and Known Issues
+* The Connectors are compatible with Tableau Desktop and Tableau Server versions 2019.4 and later unless otherwise stated.
+* As of version 2020.3, Connectors in the Extension Gallery are not compatible with Tableau Online or Tableau Prep
```

Comment:
```suggestion
* Connectors in the Extension Gallery work on Tableau Desktop and Tableau Server only.
```

We don't want to muddle the message here, and we should be explicit as to where these connectors can be used. We can change this if/when we add support for other products.